frontend/src/router.js

```diff
@@ -19,6 +19,7 @@ import SubmissionHistory from '@/components/SubmissionHistory.vue';
 import SummaryDeclaration from '@/components/SummaryDeclaration.vue';
 import SupportingDocumentUpload from '@/components/SupportingDocumentUpload.vue';
 import ApprovableFeeSchedule from '@/components/ccfriApplication/AFS/ApprovableFeeSchedule.vue';
+import CcfriClosures from '@/components/ccfriApplication/Closures.vue';
```

Comment:
This is totally fine but we usually just stick with the component name for the import definition.